integration/pom.xml

```diff
@@ -58,7 +71,9 @@
                             <goal>test</goal>
                         </goals>
                         <configuration>
+<!--                            <argLine>-Dmaven.surefire.debug -Xmx1024m ${surefire.argLine}</argLine>-->
```

Comment:
Can we delete these commented lines?

Reply:
I would like to keep them, just in case the next time will need to debug tests that are running through surefire plugin it could save time to find those arguments